src/JJMasterData.Commons/DI/JJService.cs

```diff
@@ -13,12 +14,12 @@ public static class JJService
 {
     public static IServiceProvider Provider { get; internal set; }
 
-    public static IDataAccess DataAccess
+    public static IEntityRepository EntityRepository
     {
         get
         {
             using var scope = Provider?.CreateScope();
-            return scope?.ServiceProvider.GetService<IDataAccess>() ?? new DataAccess();
+            return scope?.ServiceProvider.GetService<IEntityRepository>() ?? new Factory();
```

Comment:
IEntityRepository will never be null because its being added at `AddDefaultServices`